### PR TITLE
Overhaul site navigation

### PR DESCRIPTION
**Pull request by bacongobbler**
The guiding principle behind this PR is to move the site towards spinkube being the primary focus target.

### CODE REVIEW

File: content/en/docs/spin-trigger-mqtt/_index.md
Comment (bacongobbler) [inline content/en/docs/spin-trigger-mqtt/_index.md:1]: All of the project sections were moved over to the "SpinKube at a glance" section... Except this one. I took the liberty of removing this section from the documentation entirely. The notes here only describe how to add `spin-trigger-mqtt` to Spin, but it does not explain how to add `spin-trigger-mqtt` to SpinKube. I couldn't tell if that was possible; the docs were very unclear.

Does this seem fine?

Comment (bacongobbler) [reply content/en/docs/spin-trigger-mqtt/_index.md:1]: See https://github.com/spinkube/documentation/pull/194/files#diff-e025edb206c03227850f98095716ccef55db931b71da2f0a24d9dc91583b2ccd for more context

Comment (vdice) [reply content/en/docs/spin-trigger-mqtt/_index.md:1]: I saw your comment on the SpinKube Slack around this; let's see what the maintainers think.

**Review by vdice — APPROVED** [review]
This all looks good to me.

One Q, perhaps follow-up potential if we'd like to make the change.  Currently the `Get Started' button/link on the spinkube.dev landing page takes the reader to http://spinkube.dev/docs/overview/.  Would we rather it now deliver them to http://spinkube.dev/docs/?

**Comment by bacongobbler** [comment]
I think so, yes.

File: content/en/docs/spin-trigger-mqtt/_index.md
Comment (bacongobbler) [reply content/en/docs/spin-trigger-mqtt/_index.md:1]: Looks like the docs already exist in the project's README so this section was redundant. Merging!

**Pull request merged**